src/js/ui/disableElements.js

```diff
@@ -0,0 +1,3 @@
+export default function disableElements(...elements) {
+  [...elements].forEach(it => it.setAttribute('disabled', true));
```

Comment:
```suggestion
  elements.forEach(it => it.setAttribute('disabled', true));
```
NodeList는 forEach가 아니여도 사용할 수 있답니다!